toolkit/dump_tasks.php

```diff
@@ -28,6 +28,7 @@
 require_once(APPROOT.'core/restclient.class.inc.php');
 
 $sTaskName = Utils::ReadParameter('task_name', '*');
+Utils::$iConsoleLogLevel = Utils::ReadParameter('console_log_level', Utils::GetConfigurationValue('console_log_level', LOG_WARNING));
```

Comment:
In current code base, we have : 
`	static public $iConsoleLogLevel = LOG_INFO;`
So to my opinion this is not equivalent ?

Reply:
I took this line of code exactly from here:
https://github.com/Combodo/itop-data-collector-base/blob/60ed0c602ebdca939651eea65557a5ca51cee33d/exec.php#L81

I agree it maybe should be like this on all places?
```suggestion
Utils::$iConsoleLogLevel = Utils::ReadParameter('console_log_level', Utils::GetConfigurationValue('console_log_level', Utils::$iConsoleLogLevel));
```

Reply:
+ the `params.distrib.xml` file sets this by default to 6 (INFO), so there's no impact at all..